### DISCUSSION TITLE
My Feed - Sorting category - By Up Votes

### DISCUSSION THREAD

**Discussion by jonnyhoff** [dailydotdev/daily#562] [Feature Requests]
![image](https://user-images.githubusercontent.com/8642945/172216444-5fad131c-f334-428b-970e-9a353dfde11a.png)

**Comment by idoshamun** [comment]
Hi @jonnyhoff,

Currently, we provide the "most upvoted" feed for this purpose but we don't plan to add other sorting options to the feed at the moment.
